packages/core/src/components/icon/icon.ts

```diff
@@ -1,7 +1,10 @@
 import { Color } from '@onfido/castor';
 import { IconName } from '@onfido/castor-icons';
 
-export interface IconProps {
+export type IconProps = (Hidden | Labelled) & {
   name: IconName;
   color?: Color;
-}
+};
+
+type Hidden = { 'aria-hidden': true; 'aria-label'?: never };
+type Labelled = { 'aria-hidden'?: never; 'aria-label': string };
```

Comment:
Because we follow American English in code:

```suggestion
type Labeled = { 'aria-hidden'?: never; 'aria-label': string };
```

Source: https://www.grammarly.com/blog/labeled-labelled/

Reply:
TIL they're different